### ISSUE TITLE
19x19 features are fed twice

### ISSUE THREAD

**Issue by xonobo** [Robert-JunWang/Pelee#44]
According to dot file I obtained for the deploy_merged.prototxt the 19x19 feature maps are fed to confidence and location extraction layers twice. Is it intentional or a copy/paste error?

![pelee](https://user-images.githubusercontent.com/1115812/43190463-c48d8d7c-9001-11e8-8aa1-4deda53622b2.jpg)
[deploy_merged.pdf](https://github.com/Robert-JunWang/Pelee/files/2227207/deploy_merged.pdf)

**Comment by ujsyehao** [comment]
Yeah, it is intentional, because author removes 38x38 feature map, so use 19x19 feature map twice.

**Comment by Robert-JunWang** [comment]
It is an intentional design. But if you want to improve the performance of the small object detection, my suggestion is to use 38x38 without ResBlock instead of using 19x19 twice. The actual speed is almost the same.

**Comment by varghesealex90** [comment]
How do I pass 38x38 feature map instead of the duplicate 19x19 feature map?
